composer.json

```diff
@@ -34,16 +34,16 @@
         "ext-json": "*",
         "ext-intl": "*",
         "dompdf/dompdf": "^0.8.0",
-        "illuminate/database": "~5.5.0|~5.6.0|~5.7.0|~5.8.0",
-        "illuminate/support": "~5.5.0|~5.6.0|~5.7.0|~5.8.0",
-        "mollie/laravel-mollie": "^2.5",
+        "illuminate/database": "~5.5.0|~5.6.0|~5.7.0|~5.8.0|~6.0.0",
+        "illuminate/support": "~5.5.0|~5.6.0|~5.7.0|~5.8.0|~6.0.0",
+        "mollie/laravel-mollie": "^2.7",
         "moneyphp/money": "^3.2",
         "nesbot/carbon": "^1.33 || ^2.0"
     },
     "require-dev": {
         "mockery/mockery": "^1.0",
-        "orchestra/testbench": "~3.0",
-        "phpunit/phpunit": "^6.5|^7.5",
+        "orchestra/testbench": "~3.0|~4.0",
+        "phpunit/phpunit": "^6.5|^7.5|^8.3.4",
```

Comment:
Any reason to use `^8.3.4` instead of just `^8.3`?

Reply:
No